modules/ibi-extra-partition-ibi-install-iso.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-extra-partition-ibi-install-iso_{context}"]
+= Configuring additional partitions on the target host
+
+The installation ISO creates a partition for the `/var/lib/containers` directory as part of the image-based installation process. 
+
+You can create additional partitions by using the `coreosInstallerArgs` specification. For example, in hard disks with adequate storage, you might need an additional partition for storage options, such as {lvms-first}. 
+
+[NOTE]
+====
+The `var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images.
```

Comment:
```suggestion
The `/var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images.
```

Reply:
Do we need this note here at all?  The /var/lib/containers partition should be pre-created at this point

Reply:
I think the idea was to remind the user 500GB was required for the partition, and therefore give a hint that the start position of the new partition (`startMiB`) must be larger than that. 

Actually, I said in the callout for `startMiB` that "You must specify a starting point larger that the partition for var/lib/containers." Perhaps I can remove the note and just add the info about 500GB to the callout.